### PR TITLE
add package.json

### PR DESCRIPTION
**Pull request by kennynaoh**
Hello @glenrobertson ,
I want to add the package.json file.
Thanks. :smile: 

cc @glenrobertson

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "name": "leaflet-tilelayer-geojson",
+  "version": "1.0.4",
```
Comment (glenrobertson) [inline package.json]: version `1.0.4` doesn't exist. I only have a tag for `1.0` (https://github.com/glenrobertson/leaflet-tilelayer-geojson/releases/tag/v1.0)

can you please change it to 1.0?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  },
+  "keywords": [
+    "Leaflet",
+    "leaflet.js",
+    "tile",
+    "tilelayer",
+    "geojson"
+  ],
+  "author": "glenrobertson",
+  "license": "ISC",
```
Comment (glenrobertson) [inline package.json]: license is BSD

**Comment by glenrobertson** [comment]
Thanks for adding. I can merge after you update the comments

File: package.json (hunk 2 above)
Comment (kennynaoh) [inline package.json]: What's the detail about BSD?
[license](https://github.com/cdnjs/cdnjs/blob/master/tools/license-list.json)
Thanks.

Comment (glenrobertson) [inline package.json]: Piggybacking off of https://github.com/JasonSanford/leaflet-vector-layers/issues/63#issuecomment-218250101

Also use `"BSD-2-Clause-FreeBSD"`

**Comment by kennynaoh** [comment]
@glenrobertson 
I have updated the version and license.
Please help review it.
Thanks.

**Comment by glenrobertson** [comment]
cheers @kennynaoh looks good to me!

**Comment by kennynaoh** [comment]
Thanks a lot.

**Pull request merged**